appengine/flexible/endpoints/clients/service_to_service_non_default/appengine_config.py

```diff
@@ -0,0 +1,3 @@
+from google.appengine.ext import vendor
+
+vendor.add('path/to/google_api_python_library')
```

Comment:
We just use `lib` as the standard third-party location and instruct users to do `pip install -t lib -r requirements.txt`. You should put oauth2client in `requirements.txt`.